metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -955,4 +967,15 @@ def _get_ge_dataset(
                 **batch_kwargs,
             },
         )
+        if platform is not None and platform == "bigquery":
+            # This is done as GE makes the name as DATASET.TABLE
+            # but we want it to be PROJECT.DATASET.TABLE instead for multi-project setups
+            logger.debug(f"Setting table name to be {pretty_name}")
+            batch._table = sa.text(pretty_name)
+            name_parts = pretty_name.split(".")
+            if len(name_parts) != 3:
+                logger.error(
```

Comment:
Is it fine we just log on this error?

Reply:
Noticed there were some cases where it wasn't three for some people in OSS. Don't want to break until we understand why.